### PR TITLE
Update references section with correct links

### PR DESCRIPTION
**Pull request by YanZ777**
Remove the `.md` since that was generating an invalid URL.

### CODE REVIEW

File: docs/backend/testing/acceptance-testing-syncada-edi-invoicing.md [hunk 1]
```diff
@@ -358,5 +358,5 @@ prime-api-client --insecure support-reviewed-payment-requests --filename payload
 
 Some helpful resources for getting this setup
 
-* [Acceptance Testing Payment Requests](Acceptance-Testing-Payment-Requests.md)
-* [Manually run Prime API for Slice demo](Manually-run-Prime-API-for-Slice-demo.md)
+* [Acceptance Testing Payment Requests](Acceptance-Testing-Payment-Requests)
+* [Manually run Prime API for Slice demo](Manually-run-Prime-API-for-Slice-demo)
```
Comment (ronaktruss) [inline docs/backend/testing/acceptance-testing-syncada-edi-invoicing.md:362]: This page is located in a different directory `docs/dev/testing/running-tests/Manually-run-Prime-API-for-Slice-demo` / at this [url](https://transcom.github.io/mymove-docs/docs/dev/testing/running-tests/Manually-run-Prime-API-for-Slice-demo)

Comment (YanZ777) [reply docs/backend/testing/acceptance-testing-syncada-edi-invoicing.md:362]: I'll go check the urls again locally.

Comment (rogeruiz) [inline docs/backend/testing/acceptance-testing-syncada-edi-invoicing.md:362]:
```suggestion
* [Manually run Prime API for Slice demo](/docs/dev/testing/running-tests/Manually-run-Prime-API-for-Slice-demo)
```

**Comment by YanZ777** [comment]
Closing this PR in favor of using an [this PR](https://github.com/transcom/mymove-docs/pull/135) instead, which fixes more broken links in a better manner.

**Comment by YanZ777** [comment]
Closing this PR in favor of using an [this PR](https://github.com/transcom/mymove-docs/pull/135) instead, which fixes more broken links in a better manner.

**Pull request closed without merging**